packages/typography/src/Typography.tsx

```diff
@@ -52,6 +53,9 @@ export interface TypographyProps {
   center?: boolean
 }
 
+/**
+ * Typography
+ */
```

Comment:
Is this needed?

Reply:
Nope but it may come useful if we decide to add a description later and it's a good convention

Reply:
Makes sense